Cargo.toml

```diff
@@ -26,3 +26,4 @@ wasmparser = "0.235.0"
 [dev-dependencies]
 wasmprinter = "0.235.0"
 wat = "1.227.1"
+wasmtime = "34.0.1"
```

Comment:
I couldn't think of a good test for this which didn't require actually executing the Wasm module.